python/servo/command_base.py

```diff
@@ -938,6 +938,8 @@ def run_cargo_build_like_command(
         if with_debug_assertions or self.config["build"]["debug-assertions"]:
             env['RUSTFLAGS'] = env.get('RUSTFLAGS', "") + " -C debug_assertions"
 
+        env["RUSTC"] = "crown"
+
```

Comment:
Hrm...I don't think we should wrap the entire rust compiler. As this means we'll never be able to use cargo without mach. Isn't there another option such as writing a custom lint that could handle this?

Reply:
But we could also wrap rustc using .cargo/config.toml: https://doc.rust-lang.org/cargo/reference/config.html#buildrustc

Reply:
yep, that's what I mean by a cargo alias

Reply:
Let's use a cargo alias in that case. Will this work properly if someone was to compile Servo as a dependency of their rust project?

Reply:
Nope

Reply:
> Will this work properly if someone was to compile Servo as a dependency of their rust project?

It does not work, but it doesn't need to as crown (script_plugin) is has only static lints, it does not change codegen in any way, so this is actually improvement as downstream consumers aren't forced to run these static lints.

Reply:
Let's move this to `.cargo/config.toml` so that someday soon we can stop using `mach` to build Servo.